web/src/beta/components/fields/common/TextInput/index.tsx

```diff
@@ -2,17 +2,14 @@ import { useCallback, useEffect, useRef, useState } from "react";
 
 import { styled } from "@reearth/services/theme";
 
-import Property from "..";
-
-type Props = {
-  name?: string;
-  description?: string;
+export type Props = {
   value?: string;
+  placeholder?: string;
   timeout?: number;
   onChange?: (text: string) => void;
 };
 
-const TextInput: React.FC<Props> = ({ name, description, value, timeout = 1000, onChange }) => {
+const TextField: React.FC<Props> = ({ value, placeholder, timeout = 1000, onChange }) => {
```

Comment:
This is TextInput, and the field is TextField, right? Please revert the name to match 🙏🏻

Reply:
Done